.gitignore

```diff
@@ -1,30 +1,6 @@
-*.swp
 _build/
-setup.data
-setup.log
-setup.bin
-*.docdir
-*.native
-*.byte
-*.cmx
-*.cmi
-*.cmo
-*.o
-*.a
-*.aux
-*.log
-*.toc
-*~
-#*
-generator/_build
-generator/main.native
-ocaml/lib/datapath.ml
-ocaml/lib/plugin.ml
-ocaml/lib/volume.ml
-python/xapi/storage/api/datapath.py
-python/xapi/storage/api/plugin.py
-python/xapi/storage/api/volume.py
-ocaml/examples
-python/build
-*.pyc
-*.pyo
+*.install
+.merlin
+
```

Comment:
is the empty line here on purpose?

Reply:
Yes, I left it to separate categories of patterns (https://git-scm.com/docs/gitignore#_pattern_format), but I can remove it if you think it's ugly.

Reply:
if its on purpose its fine